packages/storybook/stories/va-combo-box-uswds.stories.jsx

```diff
@@ -46,6 +46,27 @@ const defaultArgs = {
   ],
 };
 
+const optGroupArgs = {
+  ...defaultArgs,
+  label: 'Select a fruit or vegetable',
+  options: [
+    <>
+      <optgroup label="Fruits">
+        <option value="apple">Apple</option>
+        <option value="apricot">Apricot</option>
+        <option value="cantaloupe">Cantaloupe</option>
+      </optgroup>
+      <optgroup label="Vegetables">
+        <option value="cabbage">Cabbage</option>
+        <option value="Potatoes">Potatoes</option>
+      </optgroup>
+      <option value="sweet-potatoes">Sweet Potatoes</option>
+      <option value="cherry">Cherry</option>
```

Comment:
I believe most people would be likely to associate "Cherry" and "Clementine" as "Fruits", not "Vegetables".

Reply:
In this case, `Sweet potatoes`, `Cherry`, and `Clementine` are intended to fall in neither of those two categories to demonstrate that options can be either nested or not. Maybe I just need to pick examples that are distinctively different from those categories. I also wanted to pick some non nested values outside of an `<optgroup>` that have matching segments, to demonstrate filter functionality too